src/components/dashboard/FaceVerification/components/DeviceOrientationError.js

```diff
@@ -10,8 +10,16 @@ import { withStyles } from '../../../../lib/styles'
 import FVErrorLandscapeSVG from '../../../../assets/FaceVerification/FVErrorLandscape.svg'
 
 import { fireEvent, FV_WRONGORIENTATION } from '../../../../lib/analytics/analytics'
+import { isPortrait } from '../../../../lib/utils/orientation'
+
+const SVGHeight = {
```

Comment:
Why do we need such hack ? 
- 1px difference isn't too much. It shouldn't affect design
- could we set height in % ?
- could we just use lesser value 175 in the both cases?

Reply:
@serdiukov-o-nordwhale No, we need to trigger height changes when orientation was changed. Because user first gets to this screen in landscape and all styles apply to landscape mode, and svg gets height based on landscape height (looks like small) and when we go to portrait, svg don't gets height based on portrait and still looks like small.  And if we trigger any changes in height (for now is 1 px) svg would resize again based on actual orientation. Also I think percent value for svg height it is not a solution in this case, because in some cases svg would be small or so large.